src/Data/Argonaut/Core.purs

```diff
@@ -185,8 +185,12 @@ foreign import fromBoolean :: Boolean -> Json
 -- | Construct `Json` from a `Number` value
 foreign import fromNumber :: Number -> Json
 
--- | Construct `Json` from a `String` value. If you would like to parse a string
--- | of JSON into valid `Json`, see `jsonParser`.
+-- | Construct `Json` from a `String` value.
+-- | Note that this function only produces a `Json` object containing a single piece of `String`
```

Comment:
I think this just produces a string, not an object.

Reply:
I think from JavaScript's perspective, a string is also technically an object. Really, it seems that everything is an object, which is why we can apply these instance methods:
```js
"hello".charAt(3) // "l"

z = 1.234567
z.toPrecision(3) // "1.23"
typeof z // "number"
```

However, the docs state that `String`, `Number`, etc. are not `Object` but primitives which by definition have no methods.
> a primitive (primitive value, primitive data type) is data that is **not an object and has no methods**. There are 6 primitive data types: string, number, bigint, boolean, undefined, and symbol.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures
https://developer.mozilla.org/en-US/docs/Glossary/Primitive

So I don't know what explains this inconsistency with the definitions of objects vs primitives and observed behavior. Are the docs incorrect? Is there some magic coercion/casting happening?

Back to JSON, the acronym also suggests that you always get an `object` (perhaps in the "everything is an object" sense), even if it's not key-value `Object` data (which has nothing to do with JS's "has methods" definition of `Object` - I'd rather see this named as `dict` or `map` when referring to json data). But I see how writing "object" perpetuates confusion, so I'd be up for rewriting as something like the following:
> Note that this function only produces `Json` containing

Not sure about the exact phrasing, since it sounds pretty weird if you substitute with another type, e.g. "produces `Number` containing".

In an alternate universe, we'd be working with JSAN "JavaScript Array Notation", which I'd argue is no worse than the current name.

Reply:
Yea, I'm thinking here about object in the data sense (map, dict, etc.), not object in the object-oriented programming sense. I think of JSON as being comprised of strings, numbers, booleans, arrays, objects, and null, which I believe is a common way to think of it, in which case referring to an array as an 'object' would confuse me a little.

> Note that this function only produces `Json` containing

I think this is fine phrasing.